preview/Win7Msix/Win7MSIXInstaller/ValidateTargetDeviceFamily.cpp

```diff
@@ -0,0 +1,118 @@
+#include <windows.h>
+
+#include <shlobj_core.h>
+#include <CommCtrl.h>
+
+#include "FilePaths.hpp"
+#include "ValidateTargetDeviceFamily.hpp"
+#include "GeneralUtil.hpp"
+#include <TraceLoggingProvider.h>
+#include <VersionHelpers.h>
+
+const PCWSTR ValidateTargetDeviceFamily::HandlerName = L"ValidateRequest";

```

Comment:
>ValidateRequest [](start = 57, length = 15)

change this also to ValidateTargetDeviceFamily

Reply:
Updated.